templates/Program/program.html.twig

```diff
@@ -262,4 +269,12 @@
     )
     programRecommender.init()
   </script>
+
+  <script src=" {{ asset( 'js/Translation.min.js') }}"></script>
+  <script>
+    new Translation(
+      {% if program.description %} true, {% else %} false, {% endif %}
```

Comment:
I think it would increase the readability if you just pass `program.description/credits` here and do a null/empty check inside JS.

Reply:
> However, there is no need to change it.

change needed?